docs/openapi/api-eservice-operations-interop-v1.yaml

```
@@ -266,13 +264,58 @@ paths:
                 $ref: '#/components/schemas/Problem'
               example:
                 type: string
-                status: 500
-                title: Internal Server Error
-                detail: There is a problem processing the request on the server.
-                traceId: string
-                errors:
-                - code: 000-1234
-                  detail: The server is currently down for maintenance. Please try again later.
+                status: 500 
+                title: 'Internal Server Error'
+                detail: 'There is a problem processing the request on the server.'
+                traceId: 'string'
+                errors: [{'code':'000-1234', 'detail':'The server is currently down for maintenance. Please try again later.'}]
+                
+  '/producers':
+    get:
+      summary: Retrive e-services producers
+      description: >-
+        Retrive e-services producers : producerName
```

Parameters are already defined in the specs, they should not be in the description
```suggestion
      description: Retrive e-services producers
```

---

done